web_tree_image/static/src/js/web_tree_image.js

```diff
@@ -19,7 +19,7 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-odoo.define('web.tree_image', function (require) {
+odoo.define('web.WebTreeImage', function (require) {
```

Comment:
@pedrobaeza @StefanRijnhart, What do you think of this change?

It looks ok to me, though it doesn't seem strictly necessary to port to v10